src/mysqlsh_helpers.py

```diff
@@ -56,19 +73,45 @@ def configure_mysql_users(self):
         """
         pass
 
-    def configure_instance(self):
+    def configure_instance(self) -> None:
         """Configure the instance to be used in an InnoDB cluster."""
-        pass
+        _script = f"""
+dba.configure_instance('{self.cluster_admin_user}:{self.cluster_admin_password}@{self.instance_address}')
```

Comment:
As mentioned, assemble script as tuple, then join it with:

```python
"\n".join(_script)
```

Reply:
addressed in 46822799e0a54478e31361bb4cbea6fd70c4e454